### PR TITLE
Add note to README and fix makefile

### PR DESCRIPTION
**Pull request by annoyingllama**
- added note to readme about poetry.lock dependency in data-tasks 
- updated makefike for convention
   - $() is for executing commands
   - ${} is for string substitution 
   - indentiation is done with tabs and not spaces

I confirm that I have added:

- [x ] A comment to every Dockerfile with information about target repository and tag (version).
- [ x] Filled in readme on dockerhub and in this repo about how to build the image if some extra steps should be done.

### CODE REVIEW

File: airflow/current/Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-AIRFLOW_VERSION?=2.4.1
-PYTHON_VERSION?=3.7
-COMMIT=$(shell git log -1 --pretty=%h)
-VERSION=v$(AIRFLOW_VERSION)-$(COMMIT)
-DOCKER_REPO=pubnative/airflow-2
-DOCKER_REPO_DOCKERFILE_FOLDER?=.
-DOCKER_REPO_VERSION=$(DOCKER_REPO):$(VERSION)
-DOCKER_REPO_LATEST=$(DOCKER_REPO):latest
+AIRFLOW_VERSION 				:= 2.4.1
```
Comment (Konstantinos-Chaitas-PubNative) [inline airflow/current/Makefile:1]: Can you rollback those ?

**Review by Konstantinos-Chaitas-PubNative — CHANGES_REQUESTED** [review]

File: airflow/current/Makefile (hunk 1 above)
Comment (annoyingllama) [reply airflow/current/Makefile:1]: any specific reason for using ?= instead of :=

Comment (annoyingllama) [reply airflow/current/Makefile:1]: also $() is conventionally reserved for executing commands on fly and ${} is used for string substitution. Other than that nothing is changed. My IDE goes crazy and makes everything red when its out of convention

Comment (Konstantinos-Chaitas-PubNative) [reply airflow/current/Makefile:1]: The indentation went crazy

Comment (Konstantinos-Chaitas-PubNative) [reply airflow/current/Makefile:1]: `any specific reason for using ?= instead of :=`

nope, any specific reason not to?

Comment (annoyingllama) [reply airflow/current/Makefile:1]: ?= only set the value if it is not already set. which is the not the case with us, `?=` might give the wrong idea that there is another place where you can set these values

Comment (annoyingllama) [reply airflow/current/Makefile:1]: https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting 
Section 6.5

Comment (Konstantinos-Chaitas-PubNative) [reply airflow/current/Makefile:1]: ah this is specific to Makefile, it does not exist on bash. I am good with it. Can you fix the indentation?

**Review by Konstantinos-Chaitas-PubNative — APPROVED** [review]
:rocket

File: airflow/current/Makefile (hunk 1 above)
Comment (annoyingllama) [reply airflow/current/Makefile]: fixed indentation

**Pull request merged**